### PR TITLE
remove the subtheme_videogame.style.css from assets folder (with git …

### PR DESCRIPTION
**Pull request by ilci**
…rm -f), add web/themes/custom/**/*.css to.gitignore file.
Modify some values in _variables.scss, copy page.html.twig into my templates and modify color and utility classes from light to dark. Copy navbar.twig into my components/navbar folder.

### CODE REVIEW

File: web/themes/custom/subtheme_videogame/src/components/navbar/navbar.twig [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+#}
+{{ attach_library('radix/navbar') }}
+
+{% set container = container == 'fixed' ? 'container' : false %}
+{% set placement = placement ?? '' %}
+{% set color = color ?? 'light' %}
+
+
+    <div class="{{ container }}">
+  {% endif %}
```
Comment (kittiradovics) [inline web/themes/custom/subtheme_videogame/src/components/navbar/navbar.twig]: Where's `{% if ... %}` statement?

File: web/themes/custom/subtheme_videogame/src/sass/base/_mixins.scss [hunk 2]
```diff
@@ -0,0 +1,21 @@
+$orange: #F57C00;
+$lightgrey: #EFEFEF;
+$yellow: #F3B51B;
+$darkgrey: #455A64;
+$black: #000000;
+$white: #FFFFFF;
+
```
Comment (kittiradovics) [inline web/themes/custom/subtheme_videogame/src/sass/base/_mixins.scss]: Please remove the unnecessary extra lines.

File: web/themes/custom/subtheme_videogame/src/sass/base/_mixins.scss [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+.btn-primary {
+    @include button-variant($orange, $orange, darken($lightgrey, 0%), darken($lightgrey,0%), darken($yellow,0%), darken($yellow,0%));
+}
+
+.btn-secondary {
+    @include button-variant($black, $orange, darken($darkgrey, 0%), darken($white,0%), darken($black,0%), darken($yellow,0%));
+}
+
+.navbar{
```
Comment (kittiradovics) [inline web/themes/custom/subtheme_videogame/src/sass/base/_mixins.scss]: Please add space before opening brace.

File: web/themes/custom/subtheme_videogame/src/sass/base/_mixins.scss [hunk 4]
```diff
@@ -0,0 +1,21 @@
+$orange: #F57C00;
```
Comment (kittiradovics) [inline web/themes/custom/subtheme_videogame/src/sass/base/_mixins.scss]: Please place the colors in a separate `_colors.scss` file and import it before the import of `_variables.scss`.

File: web/themes/custom/subtheme_videogame/src/components/navbar/navbar.twig [hunk 5]
```diff
@@ -0,0 +1,49 @@
+{#
```
Comment (kittiradovics) [inline web/themes/custom/subtheme_videogame/src/components/navbar/navbar.twig]: Please remove this file if haven't changed anything in it.

File: composer.lock [hunk 6]
```diff
@@ -5648,6 +5587,67 @@
             ],
             "time": "2020-01-29T20:22:20+00:00"
         },
+        {
```
Comment (kittiradovics) [inline composer.lock:5816]: Bootstrap theme should be removed. Uninstall the Bootstrap theme if you haven't yet done so (on Appearance page) and use `composer remove drupal/bootstrap`.

Comment (kittiradovics) [reply composer.lock:5816]: And please don't forget to export your configuration to enable your new theme.

File: web/themes/custom/subtheme_videogame/src/sass/base/_elements.scss [hunk 7]
```diff
@@ -54,3 +54,16 @@ strong {
 small {
 
 }
+
+.nav-link{
```
Comment (kittiradovics) [inline web/themes/custom/subtheme_videogame/src/sass/base/_elements.scss]: Please remove the unnecessary extra lines.

File: web/themes/custom/subtheme_videogame/src/sass/base/_elements.scss [hunk 8]
```diff
@@ -54,3 +54,16 @@ strong {
 small {
 
 }
+
```
Comment (kittiradovics) [inline web/themes/custom/subtheme_videogame/src/sass/base/_elements.scss]: The navbar related theming should go inside components folder.

File: web/themes/custom/subtheme_videogame/src/sass/base/_mixins.scss (hunk 3 above)
Comment (kittiradovics) [reply web/themes/custom/subtheme_videogame/src/sass/base/_mixins.scss]: This part can go into the navbar theming file.

**Review by kittiradovics — CHANGES_REQUESTED** [review]

File: web/themes/custom/subtheme_videogame/src/sass/base/_variables.scss [hunk 9]
```diff
@@ -39,8 +39,8 @@ $indigo:  #6610f2 !default;
 $purple:  #6f42c1 !default;
 $pink:    #e83e8c !default;
 $red:     #dc3545 !default;
-$orange:  #F57C00 !default;
-$yellow:  #ffc107 !default;
+$orange:  #F57C00;
```
Comment (kittiradovics) [inline web/themes/custom/subtheme_videogame/src/sass/base/_variables.scss]: As you have already created a `_colors.scss` file, you can move every color definition to that file to have every color in a single place.

File: web/themes/custom/subtheme_videogame/src/components/navbar/_navbar.scss [hunk 10]
```diff
@@ -0,0 +1,12 @@
+.nav-link{
```
Comment (kittiradovics) [inline web/themes/custom/subtheme_videogame/src/components/navbar/_navbar.scss]: Please add space before the opening curly brace.

File: web/themes/custom/subtheme_videogame/src/components/navbar/_navbar.scss [hunk 11]
```diff
@@ -0,0 +1,12 @@
...
+  text-transform: uppercase;
+  font-weight: bold;
+}
+
+.navbar-brand{
```
Comment (kittiradovics) [inline web/themes/custom/subtheme_videogame/src/components/navbar/_navbar.scss]: Please add space before the opening curly brace.

File: composer.lock [hunk 12]
```diff
@@ -1550,67 +1550,6 @@
             ],
             "time": "2019-06-08T11:03:04+00:00"
         },
-        {
-            "name": "drupal/bootstrap",
```
Comment (kittiradovics) [inline composer.lock]: `composer.json` related changes are missing.

**Review by kittiradovics — CHANGES_REQUESTED** [review]

**Review by kittiradovics — APPROVED** [review]

**Review by tamasd — APPROVED** [review]

**Pull request merged**